From 10cb31741eb9d72b643abdff33341f46e1d2913f Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 14:20:32 +0300
Subject: [PATCH 14/50] fdts: Rename clks node to plat_clks

This is needed to differentiate between SCMI clocks and
platform (hardware) clock infrastructure.

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 fdts/s32cc.dtsi                               | 128 +++++++++---------
 fdts/s32g.dtsi                                |  68 +++++-----
 fdts/s32g2.dtsi                               |  20 +--
 fdts/s32g3.dtsi                               |  90 ++++++------
 fdts/s32r45-evb.dts                           |  78 +++++------
 .../drivers/nxp/s32/clk/s32gen1_scmi_clk.h    |   2 +-
 6 files changed, 193 insertions(+), 193 deletions(-)

diff --git a/fdts/s32cc.dtsi b/fdts/s32cc.dtsi
index aa33cff3d..f0ceb61de 100644
--- a/fdts/s32cc.dtsi
+++ b/fdts/s32cc.dtsi
@@ -166,7 +166,7 @@
 		#clock-cells = <1>;
 	};
 
-	clks: clks {
+	plat_clks: plat_clks {
 		compatible = "nxp,s32cc-clocking";
 		#address-cells = <2>;
 		#size-cells = <2>;
@@ -174,13 +174,13 @@
 		status = "okay";
 
 		assigned-clocks =
-			<&clks S32GEN1_CLK_FXOSC>,
-			<&clks S32GEN1_CLK_FIRC>,
-			<&clks S32GEN1_CLK_SIRC>,
-			<&clks S32GEN1_CLK_GMAC0_EXT_RX>,
-			<&clks S32GEN1_CLK_SERDES0_LANE0_TX>,
-			<&clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
-			<&clks S32GEN1_CLK_SERDES_REF>;
+			<&plat_clks S32GEN1_CLK_FXOSC>,
+			<&plat_clks S32GEN1_CLK_FIRC>,
+			<&plat_clks S32GEN1_CLK_SIRC>,
+			<&plat_clks S32GEN1_CLK_GMAC0_EXT_RX>,
+			<&plat_clks S32GEN1_CLK_SERDES0_LANE0_TX>,
+			<&plat_clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
+			<&plat_clks S32GEN1_CLK_SERDES_REF>;
 		assigned-clock-parents =
 			<&fxosc 0>,
 			<&firc 0>,
@@ -197,11 +197,11 @@
 			reg = <0x0 0x40038000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_ARM_PLL_MUX>,
-				<&clks S32GEN1_CLK_ARM_PLL_VCO>,
-				<&clks S32GEN1_CLK_ARM_PLL_PHI0>;
+				<&plat_clks S32GEN1_CLK_ARM_PLL_MUX>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_VCO>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>;
 			assigned-clock-parents =
-				<&clks S32GEN1_CLK_FXOSC>;
+				<&plat_clks S32GEN1_CLK_FXOSC>;
 		};
 
 		armdfs: armdfs@40054000 {
@@ -209,8 +209,8 @@
 			reg = <0x0 0x40054000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_ARM_PLL_DFS1>,
-				<&clks S32GEN1_CLK_ARM_PLL_DFS2>;
+				<&plat_clks S32GEN1_CLK_ARM_PLL_DFS1>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_DFS2>;
 			assigned-clock-rates =
 				<0>,
 				<800000000>;
@@ -221,17 +221,17 @@
 			reg = <0x0 0x4003c000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_PERIPH_PLL_MUX>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_VCO>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI0>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_MUX>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_VCO>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI0>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
 			assigned-clock-parents =
-				<&clks S32GEN1_CLK_FXOSC>;
+				<&plat_clks S32GEN1_CLK_FXOSC>;
 			assigned-clock-rates =
 				<0>,
 				<S32GEN1_PERIPH_PLL_VCO_FREQ>, <100000000>,
@@ -245,8 +245,8 @@
 			reg = <0x0 0x40058000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_DFS3>;
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS3>;
 			assigned-clock-rates =
 				<S32GEN1_PERIPH_DFS1_FREQ>,
 				<S32GEN1_PERIPH_DFS3_FREQ>;
@@ -282,39 +282,39 @@
 			reg = <0x0 0x40030000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_MC_CGM0_MUX0>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX3>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX4>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX5>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX7>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX8>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX9>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX10>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX12>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX14>,
-				<&clks S32GEN1_CLK_MC_CGM0_MUX16>,
-				<&clks S32GEN1_CLK_XBAR_2X>,
-				<&clks S32GEN1_CLK_PER>,
-				<&clks S32GEN1_CLK_FTM0_REF>,
-				<&clks S32GEN1_CLK_FTM1_REF>,
-				<&clks S32GEN1_CLK_CAN_PE>,
-				<&clks S32GEN1_CLK_LIN_BAUD>,
-				<&clks S32GEN1_CLK_GMAC0_TS>,
-				<&clks S32GEN1_CLK_SPI>,
-				<&clks S32GEN1_CLK_SDHC>,
-				<&clks S32GEN1_CLK_QSPI_2X>;
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX0>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX3>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX4>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX5>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX7>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX8>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX9>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX10>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX12>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX14>,
+				<&plat_clks S32GEN1_CLK_MC_CGM0_MUX16>,
+				<&plat_clks S32GEN1_CLK_XBAR_2X>,
+				<&plat_clks S32GEN1_CLK_PER>,
+				<&plat_clks S32GEN1_CLK_FTM0_REF>,
+				<&plat_clks S32GEN1_CLK_FTM1_REF>,
+				<&plat_clks S32GEN1_CLK_CAN_PE>,
+				<&plat_clks S32GEN1_CLK_LIN_BAUD>,
+				<&plat_clks S32GEN1_CLK_GMAC0_TS>,
+				<&plat_clks S32GEN1_CLK_SPI>,
+				<&plat_clks S32GEN1_CLK_SDHC>,
+				<&plat_clks S32GEN1_CLK_QSPI_2X>;
 			assigned-clock-parents =
-				<&clks S32GEN1_CLK_ARM_PLL_DFS1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_DFS3>,
-				<&clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
+				<&plat_clks S32GEN1_CLK_ARM_PLL_DFS1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS3>,
+				<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
 			assigned-clock-rates =
 				<0>,
 				<0>,
@@ -344,10 +344,10 @@
 			reg = <0x0 0x40034000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_MC_CGM1_MUX0>,
-				<&clks S32GEN1_CLK_A53_CORE>;
+				<&plat_clks S32GEN1_CLK_MC_CGM1_MUX0>,
+				<&plat_clks S32GEN1_CLK_A53_CORE>;
 			assigned-clock-parents =
-				<&clks S32GEN1_CLK_ARM_PLL_PHI0>,
+				<&plat_clks S32GEN1_CLK_ARM_PLL_PHI0>,
 				<0>;
 		};
 
@@ -356,10 +356,10 @@
 			reg = <0x0 0x40068000 0x0 0x3000>;
 
 			assigned-clocks =
-				<&clks S32GEN1_CLK_MC_CGM5_MUX0>,
-				<&clks S32GEN1_CLK_DDR>;
+				<&plat_clks S32GEN1_CLK_MC_CGM5_MUX0>,
+				<&plat_clks S32GEN1_CLK_DDR>;
 			assigned-clock-parents =
-				<&clks S32GEN1_CLK_DDR_PLL_PHI0>,
+				<&plat_clks S32GEN1_CLK_DDR_PLL_PHI0>,
 				<0>;
 			assigned-clock-rates =
 				<0>,
diff --git a/fdts/s32g.dtsi b/fdts/s32g.dtsi
index 220c65fae..c75197151 100644
--- a/fdts/s32g.dtsi
+++ b/fdts/s32g.dtsi
@@ -12,34 +12,34 @@
 	model = "NXP S32G";
 };
 
-&clks {
-	clocks = <&clks S32GEN1_CLK_PER>,
-		<&clks S32GEN1_CLK_FTM0_REF>,
-		<&clks S32GEN1_CLK_FTM1_REF>,
-		<&clks S32GEN1_CLK_CAN_PE>,
-		<&clks S32GEN1_CLK_XBAR_2X>,
-		<&clks S32GEN1_CLK_XBAR>,
-		<&clks S32GEN1_CLK_XBAR_DIV2>,
-		<&clks S32GEN1_CLK_XBAR_DIV3>,
-		<&clks S32GEN1_CLK_XBAR_DIV4>,
-		<&clks S32GEN1_CLK_XBAR_DIV6>,
-		<&clks S32GEN1_CLK_SPI>,
-		<&clks S32GEN1_CLK_QSPI>;
+&plat_clks {
+	clocks = <&plat_clks S32GEN1_CLK_PER>,
+		<&plat_clks S32GEN1_CLK_FTM0_REF>,
+		<&plat_clks S32GEN1_CLK_FTM1_REF>,
+		<&plat_clks S32GEN1_CLK_CAN_PE>,
+		<&plat_clks S32GEN1_CLK_XBAR_2X>,
+		<&plat_clks S32GEN1_CLK_XBAR>,
+		<&plat_clks S32GEN1_CLK_XBAR_DIV2>,
+		<&plat_clks S32GEN1_CLK_XBAR_DIV3>,
+		<&plat_clks S32GEN1_CLK_XBAR_DIV4>,
+		<&plat_clks S32GEN1_CLK_XBAR_DIV6>,
+		<&plat_clks S32GEN1_CLK_SPI>,
+		<&plat_clks S32GEN1_CLK_QSPI>;
 
 	assigned-clocks =
-		<&clks S32GEN1_CLK_FXOSC>,
-		<&clks S32GEN1_CLK_FIRC>,
-		<&clks S32GEN1_CLK_SIRC>,
-		<&clks S32GEN1_CLK_GMAC0_EXT_RX>,
-		<&clks S32G_CLK_SERDES1_LANE0_TX>,
-		<&clks S32G_CLK_SERDES1_LANE0_CDR>,
-		<&clks S32G_CLK_SERDES1_LANE1_TX>,
-		<&clks S32G_CLK_SERDES1_LANE1_CDR>,
-		<&clks S32GEN1_CLK_SERDES0_LANE0_TX>,
-		<&clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
-		<&clks S32G_CLK_SERDES0_LANE1_TX>,
-		<&clks S32G_CLK_SERDES0_LANE1_CDR>,
-		<&clks S32GEN1_CLK_SERDES_REF>;
+		<&plat_clks S32GEN1_CLK_FXOSC>,
+		<&plat_clks S32GEN1_CLK_FIRC>,
+		<&plat_clks S32GEN1_CLK_SIRC>,
+		<&plat_clks S32GEN1_CLK_GMAC0_EXT_RX>,
+		<&plat_clks S32G_CLK_SERDES1_LANE0_TX>,
+		<&plat_clks S32G_CLK_SERDES1_LANE0_CDR>,
+		<&plat_clks S32G_CLK_SERDES1_LANE1_TX>,
+		<&plat_clks S32G_CLK_SERDES1_LANE1_CDR>,
+		<&plat_clks S32GEN1_CLK_SERDES0_LANE0_TX>,
+		<&plat_clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
+		<&plat_clks S32G_CLK_SERDES0_LANE1_TX>,
+		<&plat_clks S32G_CLK_SERDES0_LANE1_CDR>,
+		<&plat_clks S32GEN1_CLK_SERDES_REF>;
 	assigned-clock-parents =
 		<&fxosc 0>,
 		<&firc 0>,
@@ -64,24 +64,24 @@
 		reg = <0x0 0x44018000 0x0 0x3000>;
 
 		assigned-clocks =
-			<&clks S32G_CLK_MC_CGM2_MUX0>,
-			<&clks S32G_CLK_PFE_PE>;
+			<&plat_clks S32G_CLK_MC_CGM2_MUX0>,
+			<&plat_clks S32G_CLK_PFE_PE>;
 		assigned-clock-rates =
 			<0>,
 			<600000000>;
 		assigned-clock-parents =
-			<&clks S32G_CLK_ACCEL_PLL_PHI1>;
+			<&plat_clks S32G_CLK_ACCEL_PLL_PHI1>;
 	};
 };
 
 &accelpll {
 	assigned-clocks =
-		<&clks S32GEN1_CLK_ACCEL_PLL_MUX>,
-		<&clks S32GEN1_CLK_ACCEL_PLL_VCO>,
-		<&clks S32G_CLK_ACCEL_PLL_PHI0>,
-		<&clks S32G_CLK_ACCEL_PLL_PHI1>;
+		<&plat_clks S32GEN1_CLK_ACCEL_PLL_MUX>,
+		<&plat_clks S32GEN1_CLK_ACCEL_PLL_VCO>,
+		<&plat_clks S32G_CLK_ACCEL_PLL_PHI0>,
+		<&plat_clks S32G_CLK_ACCEL_PLL_PHI1>;
 	assigned-clock-parents =
-		<&clks S32GEN1_CLK_FXOSC>;
+		<&plat_clks S32GEN1_CLK_FXOSC>;
 	assigned-clock-rates =
 		<0>,
 		<1800000000>,
diff --git a/fdts/s32g2.dtsi b/fdts/s32g2.dtsi
index ebfea7e86..69cebd093 100644
--- a/fdts/s32g2.dtsi
+++ b/fdts/s32g2.dtsi
@@ -23,16 +23,16 @@
 };
 
 &mc_cgm0 {
-	clocks = <&clks S32GEN1_CLK_XBAR_2X>,
-		<&clks S32GEN1_CLK_SERDES_REF>,
-		<&clks S32GEN1_CLK_PER>,
-		<&clks S32GEN1_CLK_FTM0_REF>,
-		<&clks S32GEN1_CLK_FTM1_REF>,
-		<&clks S32GEN1_CLK_CAN_PE>,
-		<&clks S32GEN1_CLK_LIN_BAUD>,
-		<&clks S32GEN1_CLK_SPI>,
-		<&clks S32GEN1_CLK_SDHC>,
-		<&clks S32GEN1_CLK_QSPI_2X>;
+	clocks = <&plat_clks S32GEN1_CLK_XBAR_2X>,
+		<&plat_clks S32GEN1_CLK_SERDES_REF>,
+		<&plat_clks S32GEN1_CLK_PER>,
+		<&plat_clks S32GEN1_CLK_FTM0_REF>,
+		<&plat_clks S32GEN1_CLK_FTM1_REF>,
+		<&plat_clks S32GEN1_CLK_CAN_PE>,
+		<&plat_clks S32GEN1_CLK_LIN_BAUD>,
+		<&plat_clks S32GEN1_CLK_SPI>,
+		<&plat_clks S32GEN1_CLK_SDHC>,
+		<&plat_clks S32GEN1_CLK_QSPI_2X>;
 };
 
 &ocotp {
diff --git a/fdts/s32g3.dtsi b/fdts/s32g3.dtsi
index 985e078f1..61f73e745 100644
--- a/fdts/s32g3.dtsi
+++ b/fdts/s32g3.dtsi
@@ -12,41 +12,41 @@
 	compatible = "nxp,s32g3";
 };
 
-&clks {
+&plat_clks {
 
 	mc_cgm0: mc_cgm0@40030000 {
 		compatible = "nxp,s32cc-mc_cgm0";
 		reg = <0x0 0x40030000 0x0 0x3000>;
 
 		assigned-clocks =
-			<&clks S32GEN1_CLK_MC_CGM0_MUX0>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX3>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX4>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX5>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX7>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX8>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX12>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX14>,
-			<&clks S32GEN1_CLK_MC_CGM0_MUX16>,
-			<&clks S32GEN1_CLK_PER>,
-			<&clks S32GEN1_CLK_XBAR_2X>,
-			<&clks S32GEN1_CLK_FTM0_REF>,
-			<&clks S32GEN1_CLK_FTM1_REF>,
-			<&clks S32GEN1_CLK_CAN_PE>,
-			<&clks S32GEN1_CLK_LIN_BAUD>,
-			<&clks S32GEN1_CLK_SPI>,
-			<&clks S32GEN1_CLK_QSPI_2X>,
-			<&clks S32GEN1_CLK_SDHC>;
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX0>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX3>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX4>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX5>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX7>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX8>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX12>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX14>,
+			<&plat_clks S32GEN1_CLK_MC_CGM0_MUX16>,
+			<&plat_clks S32GEN1_CLK_PER>,
+			<&plat_clks S32GEN1_CLK_XBAR_2X>,
+			<&plat_clks S32GEN1_CLK_FTM0_REF>,
+			<&plat_clks S32GEN1_CLK_FTM1_REF>,
+			<&plat_clks S32GEN1_CLK_CAN_PE>,
+			<&plat_clks S32GEN1_CLK_LIN_BAUD>,
+			<&plat_clks S32GEN1_CLK_SPI>,
+			<&plat_clks S32GEN1_CLK_QSPI_2X>,
+			<&plat_clks S32GEN1_CLK_SDHC>;
 		assigned-clock-parents =
-			<&clks S32GEN1_CLK_ARM_PLL_DFS1>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_DFS3>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
+			<&plat_clks S32GEN1_CLK_ARM_PLL_DFS1>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI1>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI2>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI3>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS1>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_DFS3>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI7>;
 		assigned-clock-rates =
 			<0>,
 			<0>,
@@ -67,16 +67,16 @@
 			<400000000>,
 			<400000000>;
 
-		clocks = <&clks S32GEN1_CLK_XBAR_2X>,
-			<&clks S32GEN1_CLK_SERDES_REF>,
-			<&clks S32GEN1_CLK_PER>,
-			<&clks S32GEN1_CLK_FTM0_REF>,
-			<&clks S32GEN1_CLK_FTM1_REF>,
-			<&clks S32GEN1_CLK_CAN_PE>,
-			<&clks S32GEN1_CLK_LIN_BAUD>,
-			<&clks S32GEN1_CLK_SPI>,
-			<&clks S32GEN1_CLK_SDHC>,
-			<&clks S32GEN1_CLK_QSPI_2X>;
+		clocks = <&plat_clks S32GEN1_CLK_XBAR_2X>,
+			<&plat_clks S32GEN1_CLK_SERDES_REF>,
+			<&plat_clks S32GEN1_CLK_PER>,
+			<&plat_clks S32GEN1_CLK_FTM0_REF>,
+			<&plat_clks S32GEN1_CLK_FTM1_REF>,
+			<&plat_clks S32GEN1_CLK_CAN_PE>,
+			<&plat_clks S32GEN1_CLK_LIN_BAUD>,
+			<&plat_clks S32GEN1_CLK_SPI>,
+			<&plat_clks S32GEN1_CLK_SDHC>,
+			<&plat_clks S32GEN1_CLK_QSPI_2X>;
 	};
 
 	mc_cgm6: mc_cgm0@4053c000 {
@@ -84,14 +84,14 @@
 		reg = <0x0 0x4053c000 0x0 0x3000>;
 
 		assigned-clocks =
-			<&clks S32G_CLK_MC_CGM6_MUX0>,
-			<&clks S32G_CLK_MC_CGM6_MUX1>,
-			<&clks S32G_CLK_MC_CGM6_MUX2>,
-			<&clks S32GEN1_CLK_GMAC0_TS>;
+			<&plat_clks S32G_CLK_MC_CGM6_MUX0>,
+			<&plat_clks S32G_CLK_MC_CGM6_MUX1>,
+			<&plat_clks S32G_CLK_MC_CGM6_MUX2>,
+			<&plat_clks S32GEN1_CLK_GMAC0_TS>;
 		assigned-clock-parents =
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
-			<&clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
-			<&clks S32GEN1_CLK_GMAC0_EXT_RX>;
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI4>,
+			<&plat_clks S32GEN1_CLK_PERIPH_PLL_PHI5>,
+			<&plat_clks S32GEN1_CLK_GMAC0_EXT_RX>;
 		assigned-clock-rates =
 			<0>,
 			<0>,
diff --git a/fdts/s32r45-evb.dts b/fdts/s32r45-evb.dts
index 462921bf9..b598ba712 100644
--- a/fdts/s32r45-evb.dts
+++ b/fdts/s32r45-evb.dts
@@ -41,26 +41,26 @@
 	};
 };
 
-&clks {
-	clocks = <&clks S32GEN1_CLK_PER>,
-		<&clks S32GEN1_CLK_FTM0_REF>,
-		<&clks S32GEN1_CLK_FTM1_REF>,
-		<&clks S32GEN1_CLK_CAN_PE>,
-		<&clks S32GEN1_CLK_SPI>,
-		<&clks S32GEN1_CLK_QSPI>,
-		<&clks S32R45_CLK_ACCEL4>,
-		<&clks S32R45_CLK_ACCEL3>;
+&plat_clks {
+	clocks = <&plat_clks S32GEN1_CLK_PER>,
+		<&plat_clks S32GEN1_CLK_FTM0_REF>,
+		<&plat_clks S32GEN1_CLK_FTM1_REF>,
+		<&plat_clks S32GEN1_CLK_CAN_PE>,
+		<&plat_clks S32GEN1_CLK_SPI>,
+		<&plat_clks S32GEN1_CLK_QSPI>,
+		<&plat_clks S32R45_CLK_ACCEL4>,
+		<&plat_clks S32R45_CLK_ACCEL3>;
 
 	assigned-clocks =
-			<&clks S32GEN1_CLK_FXOSC>,
-			<&clks S32GEN1_CLK_FIRC>,
-			<&clks S32GEN1_CLK_SIRC>,
-			<&clks S32GEN1_CLK_GMAC0_EXT_RX>,
-			<&clks S32R45_CLK_SERDES1_LANE0_TX>,
-			<&clks S32R45_CLK_SERDES1_LANE0_CDR>,
-			<&clks S32GEN1_CLK_SERDES0_LANE0_TX>,
-			<&clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
-			<&clks S32GEN1_CLK_SERDES_REF>;
+			<&plat_clks S32GEN1_CLK_FXOSC>,
+			<&plat_clks S32GEN1_CLK_FIRC>,
+			<&plat_clks S32GEN1_CLK_SIRC>,
+			<&plat_clks S32GEN1_CLK_GMAC0_EXT_RX>,
+			<&plat_clks S32R45_CLK_SERDES1_LANE0_TX>,
+			<&plat_clks S32R45_CLK_SERDES1_LANE0_CDR>,
+			<&plat_clks S32GEN1_CLK_SERDES0_LANE0_TX>,
+			<&plat_clks S32GEN1_CLK_SERDES0_LANE0_CDR>,
+			<&plat_clks S32GEN1_CLK_SERDES_REF>;
 	assigned-clock-parents =
 			<&fxosc 0>,
 			<&firc 0>,
@@ -80,13 +80,13 @@
 		reg = <0x0 0x440c0000 0x0 0x3000>;
 
 		assigned-clocks =
-			<&clks S32R45_CLK_MC_CGM2_MUX0>,
-			<&clks S32R45_CLK_MC_CGM2_MUX1>,
-			<&clks S32R45_CLK_ACCEL3>,
-			<&clks S32R45_CLK_ACCEL4>;
+			<&plat_clks S32R45_CLK_MC_CGM2_MUX0>,
+			<&plat_clks S32R45_CLK_MC_CGM2_MUX1>,
+			<&plat_clks S32R45_CLK_ACCEL3>,
+			<&plat_clks S32R45_CLK_ACCEL4>;
 		assigned-clock-parents =
-			<&clks S32R45_CLK_ACCEL_PLL_PHI0>,
-			<&clks S32R45_CLK_ARM_PLL_DFS4_2>;
+			<&plat_clks S32R45_CLK_ACCEL_PLL_PHI0>,
+			<&plat_clks S32R45_CLK_ARM_PLL_DFS4_2>;
 		assigned-clock-rates =
 			<0>,
 			<0>,
@@ -97,12 +97,12 @@
 
 &accelpll {
 	assigned-clocks =
-		<&clks S32GEN1_CLK_ACCEL_PLL_MUX>,
-		<&clks S32GEN1_CLK_ACCEL_PLL_VCO>,
-		<&clks S32R45_CLK_ACCEL_PLL_PHI0>,
-		<&clks S32GEN1_CLK_ARM_PLL_DFS4>;
+		<&plat_clks S32GEN1_CLK_ACCEL_PLL_MUX>,
+		<&plat_clks S32GEN1_CLK_ACCEL_PLL_VCO>,
+		<&plat_clks S32R45_CLK_ACCEL_PLL_PHI0>,
+		<&plat_clks S32GEN1_CLK_ARM_PLL_DFS4>;
 	assigned-clock-parents =
-		<&clks S32GEN1_CLK_FXOSC>;
+		<&plat_clks S32GEN1_CLK_FXOSC>;
 	assigned-clock-rates =
 		<0>,
 		<1800000000>,
@@ -112,19 +112,19 @@
 
 
 &mc_cgm0 {
-	clocks = <&clks S32GEN1_CLK_SERDES_REF>,
-		<&clks S32GEN1_CLK_PER>,
-		<&clks S32GEN1_CLK_FTM0_REF>,
-		<&clks S32GEN1_CLK_FTM1_REF>,
-		<&clks S32GEN1_CLK_CAN_PE>,
-		<&clks S32GEN1_CLK_LIN_BAUD>,
-		<&clks S32GEN1_CLK_SPI>,
-		<&clks S32GEN1_CLK_QSPI>;
+	clocks = <&plat_clks S32GEN1_CLK_SERDES_REF>,
+		<&plat_clks S32GEN1_CLK_PER>,
+		<&plat_clks S32GEN1_CLK_FTM0_REF>,
+		<&plat_clks S32GEN1_CLK_FTM1_REF>,
+		<&plat_clks S32GEN1_CLK_CAN_PE>,
+		<&plat_clks S32GEN1_CLK_LIN_BAUD>,
+		<&plat_clks S32GEN1_CLK_SPI>,
+		<&plat_clks S32GEN1_CLK_QSPI>;
 };
 
 &mc_cgm2 {
-	clocks = <&clks S32R45_CLK_ACCEL4>,
-		<&clks S32R45_CLK_ACCEL3>;
+	clocks = <&plat_clks S32R45_CLK_ACCEL4>,
+		<&plat_clks S32R45_CLK_ACCEL3>;
 };
 
 &ddr_errata {
diff --git a/include/drivers/nxp/s32/clk/s32gen1_scmi_clk.h b/include/drivers/nxp/s32/clk/s32gen1_scmi_clk.h
index 992a56a20..ffc46cb24 100644
--- a/include/drivers/nxp/s32/clk/s32gen1_scmi_clk.h
+++ b/include/drivers/nxp/s32/clk/s32gen1_scmi_clk.h
@@ -9,7 +9,7 @@
 #include <stdint.h>
 #include <stdbool.h>
 
-#define S32GEN1_CLK_DRV_NAME	"clks"
+#define S32GEN1_CLK_DRV_NAME	"plat_clks"
 
 #define SCMI_ARRAY_ENTRY(ID, PLAT_ID, NAME) \
 	[INDEX(ID)] = { .plat_id = (PLAT_ID), .name = (NAME), }
-- 
2.17.1

